src/components/Atoms/Form/RadioField.tsx

```diff
@@ -81,21 +81,26 @@ export default function RadioField(props: RadioFieldProps) {
                   : 'input-border-normal ui-not-disabled:focus-visible:ring-blue-faded/50'
               }`}
               >
-                {({ checked }) => (
-                  <span
-                    className={`block h-full w-full select-none py-1.5 px-3 text-center ${
-                      disabledVisual
-                        ? checked
-                          ? 'bg-blue-faded/30 text-white/50'
-                          : 'bg-gray-800 text-white/50'
-                        : checked
-                        ? 'bg-[#2B4554] text-gray-300'
-                        : 'text-white/75'
-                    }`}
-                  >
-                    {option}
-                  </span>
-                )}
+                {({ checked }) => {
+                  const initialChecked =
+                    selectedValue !== undefined &&
```

Comment:
I thought selectedValue is "false" if it's undefined. Thus it can never be undefined (line18)